utils/docker/Dockerfile.checkpatch

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2021 Intel Corporation
+# All rights reserved.
+#
+# 'recipe' for Docker to build a image for running static analysis checking.
+#
+
+# Pull base image
+FROM fedora:33
```

Comment:
Didn't we discuss using `:latest` here so that it's not somebody's job to have (to remember) to fix this every 6 months?

Reply:
I can do whichever you prefer, but I think we need to add --pull to the Jenkinsfile if we do that.